manifests/k/KhronosGroup/VulkanSDK/1.4.304.1/KhronosGroup.VulkanSDK.locale.en-US.yaml

```diff
@@ -11,14 +11,14 @@ PrivacyUrl: https://vulkan.lunarg.com/sdk/home#content/popup/privacy/lg
 PackageName: Vulkan SDK
 PackageUrl: https://vulkan.lunarg.com/sdk/home
 License: MIT
-LicenseUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.296.0-windows-license-summary.txt
+LicenseUrl: https://vulkan.lunarg.com/software/license/vulkan-1.4.304.1-windows-license-summary.txt
 Copyright: Copyright (c) 2015-2022 LunarG, Inc.
-CopyrightUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.296.0-windows-license-summary.txt
+CopyrightUrl: https://vulkan.lunarg.com/software/license/vulkan-1.4.304.1-windows-license-summary.txt

```

Comment:
This file has a long list of copyrights, and I'm unsure if we should update the one with a more recent date, these all seem to be pulled from code with wild formatting.

Reply:
Hi @MarijnS95, 

Unfortunately, you're probably the most informed regarding this topic, so you would probably be the best person to make this decision. Generally, a newer copyright is better, but you would know more here than I do.